utilities_common/bgp_util.py

```diff
@@ -153,8 +153,11 @@ def run_bgp_command(vtysh_cmd, bgp_namespace=multi_asic.DEFAULT_NAMESPACE):
     if bgp_namespace is not multi_asic.DEFAULT_NAMESPACE:
         bgp_instance_id = " -n {} ".format(multi_asic.get_asic_id_from_name(bgp_namespace))
 
-    cmd = 'sudo vtysh {} -c "{}"'.format(
-        bgp_instance_id, vtysh_cmd)
+    vtysh = 'vtysh'
+    if vtysh_cmd.startswith('show'):
+        vtysh = 'rvtysh'
```

Comment:
>rvtysh [](http://example.com/codeflow?start=17&length=6)

Let's extract it into a string constant into `sonic-py-common`. The name is arguable. How about `vtysh_show`?

Reply:
Fixed, changed to use the variable name vtysh_shell_cmd, and added the constants.